tree/dataframe/test/dataframe_snapshotWithVariations.cxx

```diff
@@ -0,0 +1,435 @@
+#include <ROOT/RDataFrame.hxx>
+#include "TwoFloats.h"
+
+#include <TBranch.h>
+#include <TDataType.h>
+#include <TFile.h>
+#include <TInterpreter.h>
+#include <TLeaf.h>
+#include <TTree.h>
+#include <TSystem.h>
+
+#include <gtest/gtest.h>
+
+#include <bitset>
+#include <numeric>
+
+constexpr bool verbose = false;
```

Comment:
How can this be controlled? Is it something useful only when changing the file locally?

Reply:
It's activating code that will print the trees, and this is I believe useful if anybody would like to debug the tests. I would like to leave it there (but disabled) so that future maintainers don't need to rewrite the code.
The constexpr is the best way to do it, because we get syntax checking, but no actual code is emitted. It's similar to what the kernel does with `if (false) {`.